admin/installation.rst

```diff
@@ -532,6 +532,148 @@ DNF provide simple way to add repository:
 Once added, you can install any package with ``dnf install`` (e.g. ``dnf install netxms-agent``).
 
 
+Installing packages
+-------------------
+
+Server
+~~~~~~
+
+Server require two components to function - server itself (package "netxms-server") and at least one database abstraction layer driver 
+(multiple can be installed at the same time, e.g. for migration purposes). These database drivers are also used by agent for database 
+monitoring (performing queries to databases). 
+
+Provided driver packages:
+
+  * netxms-dbdrv-pgsql - PostgreSQL driver
+  * netxms-dbdrv-mariadb - Mariadb driver
+  * netxms-dbdrv-mysql - MySQL driver, currently under development (not built for Ubuntu 20 / Mint 20)
+  * netxms-dbdrv-odbc - unixODBC driver (can be used with DB/2 and Microsoft SQL)
+  * netxms-dbdrv-oracle - Oracle driver ( requires Oracle client installation )
+
+#. Instal required packages (adjust command to match your environment):
+
+   .. code-block:: sh
+
+     dnf install netxms-server netxms-dbdrv-pgsql
+
+#. Create user and database (:ref:`examples <db_creation>`).
+
+#. Modify server configuration file ("/etc/netxmsd.conf" to match your environment.
+
+#. Load database schema and default configuration:
+
+   .. code-block:: sh
+
+     nxdbmgr init
+
+#. Start server:
+
+   .. code-block:: sh
+
+     systemctl start netxms-server.service
+
+#. Enable automatic startup of server:
+
+   .. code-block:: sh
+
+     systemctl enable netxms-server.service
+
+#. If database engine is running on the same system, add ordering dependency for
+   database into netxmsd systemd unit override file. This will ensure database
+   shutdown only after netxmsd process completion on system shutdown/restart. To
+   add the dependency e.g. for Postgres database, run:
+
+   .. code-block:: sh
+
+     systemctl edit netxmsd
+   
+   and add the following lines:
+
+   .. code-block:: sh
+
+     [Unit]
+     After=network.target postgresql.service
+
+   After editing run ``systemctl daemon-reload`` to reload systemd
+   configuration. 
+
+.. note::
+
+  Default credentials - user "admin" with password "netxms".
+
+
+Agent
+~~~~~
+
+Install core agent package ("netxms-agent") and optional subagent packages, if required:
+
+.. code-block:: sh
+
+  apt-get install netxms-agent
+
+Start agent
+
+.. code-block:: sh
+
+  systemctl start netxms-agent
+
+Enable automatic startup of agent
+
+.. code-block:: sh
+
+  systemctl enable netxms-agent
+
+
+Management Client
+~~~~~~~~~~~~~~~~~
+
+Desktop Management Client
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Due to limitation of Eclipse platform used to build the Management Client, only x64 build is provided.
+
+ 1. Make sure you have 64-bit Java version 17 installed you your system. 
+ 
+ 2. Download the latest version from http://www.netxms.org/download. You will need
```

Comment:
This should be updated to current situation:
- download .jar file
- run it using java -jar nxmc-xxx.jar